components/CheckInOutForm/index.tsx

```diff
@@ -231,6 +264,28 @@ function CheckInOutForm({
         value={aaSelected}
         setValue={setAaSelected}
       />
+      {/* Text Fields */}
+      {splitAttrs.text.map((textAttr) => (
+        <TextField
+          key={textAttr._id}
+          label={textAttr.name}
+          onChange={(_e) => updateTextFieldAttributes(_e, textAttr._id)}
```

Comment:
_ as a prefix is a convention for unused (discard) variables. In the past, we've needed it to be able to access the second argument generated by `onChange`. Here, though, we are actually using it. Rename this and all other instances of `_e` to just `e`. Nit-picky, but it helps the programmer know how things are being used.

Reply:
Also, the event doesn't know what type the `event.target.value` is, so we must specify what it is based on the TextField type. Change this `onChange` to pass in `e.target.value`. This works in conjunction with the change to the `updateTextFieldAttributes` prototype in another comment.